README.md

```diff
@@ -1,11 +1,53 @@
 [![Build Status](https://travis-ci.com/brain-score/brain-score.web.svg?branch=master)](https://travis-ci.com/brain-score/brain-score.web)
 
 ## Setup
-Install dependencies: `pip install .`
+
+Ensure you are using `python@3.8`
```

Comment:
Does anything break from 3.7 -> 3.8? I have been using 3.7 this whole time, but does the new changes in the UI require 3.8?

Reply:
ha no, tbh I thought you were on 3.8 that's why I picked that version. I can change this to `<= python@3.8`